### PR TITLE
Filter and aggregationMethod examples in Property docs

### PR DESCRIPTION
**Pull request by teallarson**
- Added a list of common filters and aggregationMethods, though those vary by source (also noted that and where to find what ones they can use)
- Added two code blocks with examples of how to combine aggregationMethods and filters to create more complex properties

### CODE REVIEW

File: pages/docs/config/properties/community.mdx [hunk 1]
```diff
@@ -96,4 +96,107 @@ The generator attempts to make a _reasonable_ guess at the values in the config
 
 Other sections may be commented out but require some action by you. The best practice when configuring an Property is to read the comments, keys, and values within the generated file and fill in the appropriate values for your Property.
 
+### Aggregation Method and Filters
+
+Your Property config file will also contain `aggregationMethod` and `filter` options. Options for these may vary depending on your source type. All available options for your source will be listed in your config file template. Some common examples are:
```
Comment (evantahler) [inline pages/docs/config/properties/community.mdx]: Maybe a little more info about where to get a template:

> All available options for your source will be listed in your config file template created when you [`grouparoo generate`](/docs/cli/config#generate). Some common examples are:

**Review by evantahler — COMMENTED** [review]
Looks good! Just a non-blocking suggestion below

File: pages/docs/config/properties/community.mdx (hunk 1 above)
Comment (teallarson) [reply pages/docs/config/properties/community.mdx]: Sounds good.  Added that.

**Review by evantahler — APPROVED** [review]
Please make that new copy a link to /docs/cli/config#generate and this is good to go!

**Pull request merged**